extension/test/data_test.ts

```diff
@@ -69,4 +69,60 @@ describe('data.ts', function () {
       ).to.include.something.like({ entry: /^ＤＶＤ-ＲＯＭ .*/ });
     });
   });
+
+  describe('makeText', function () {
+    it('should return the text for a word dictionary entry', function () {
+      const entry = rcxDict.wordSearch('あ', /* doNames= */ false);
```

Comment:
I think ideally we'd construct the entry our selves instead of relying on `wordSearch` this would allow us to explicitly see what parts of the entry are important to the output. Right now, the reader needs to understand how `wordSearch` works to know if the test is correct and why.

It also makes the test  less precise since a bug in `wordSearch` will cause the test to fail.

That said, this is still a clear improvement so feel free to keep like this and just remember the overall idea.

Reply:
Yeah, I agree - good point. I want these makeText tests to only fail if the makeText function fails to adhere to its specification, but the way I wrote this test makes it possible for wordSearch to cause this test to fail, which makes the tests harder to understand when they fail.